bluetooth: controller: New release of ble_controller library (#131)
NordicPlayground/nrfxlib: ble_controller/CHANGELOG.rst

Changes
=======

* :cpp:func:`mpsl_init()` is no longer called by :cpp:func:`ble_controller_init()`.
  Application is therefore responsible for calling :cpp:func:`mpsl_init()`, and it must be done before :cpp:func:`ble_controller_init()` is called.
* Clock configuration parameters are now contained in :c:type:`mpsl_clock_lfclk_cfg_t` instead of :c:type:`nrf_lf_clock_cfg_t`,
  and must be provided to :cpp:func:`mpsl_init()` instead of :cpp:func:`ble_controller_init()`.
* Clock accuracy must now be specified in ppm instead of the previous enum value.
* The IRQ line to pend for low priority signal processing must be provided to :cpp:func:`mpsl_init()` instead of :cpp:func:`ble_controller_init()`.
* The application must call :cpp:func:`mpsl_low_priority_process()` instead of :cpp:func:`ble_controller_low_prio_tasks_process()` to process low priority signals.
* :cpp:func:`mpsl_uninit()` is no longer called by :cpp:func:`ble_controller_disable()`.
  Application must therefore call :cpp:func:`mpsl_uninit()` after :cpp:func:`ble_controller_disable()` to uninitialize MPSL.
* Interrupt handler APIs for the following peripherals are moved to MPSL: RADIO, RTC0, TIMER0 and POWER_CLOCK.

> * High frequency clock API (``ble_controller_hf_clock_...``) is removed. Use corresponding API in MPSL instead.

Reviewer: ```suggestion
* High frequency clock API (``ble_controller_hf_clock_...``) is removed.
  Use the corresponding API in MPSL instead.
```

> * Temperature API (:cpp:func:`ble_controller_temp_get()`) is removed. Use corresponding API in MPSL instead.

Reviewer: ```suggestion
* Temperature API (:cpp:func:`ble_controller_temp_get()`) is removed.
  Use the corresponding API in MPSL instead.
```

> * Timeslot API is removed. Use corresponding API in MPSL instead.

Reviewer: ```suggestion
* Timeslot API is removed.
  Use the corresponding API in MPSL instead.
```

* Version numbers have been removed from the libraries.

Bugfixes
========

* Fixed an issue on nRF53 where the T_IFS in certain conditions was off by 5 us.
* Fixed an issue where a control packet could be sent twice even after the packet was ACKed.

>   This could only occur if the radio was forced off due to a unforeseen condition.

Reviewer: ```suggestion
  This would only occur if the radio was forced off due to an unforeseen condition.
```
